lib/matplotlib/backend_managers.py

```diff
@@ -79,14 +82,15 @@ def __init__(self, figure, num):
         self.window = self._backend.Window('Figure %d' % num)
         self.window.mpl_connect('window_destroy_event', self.destroy)
 
-        self.canvas = self._backend.FigureCanvas(figure, manager=self)
+        self._figure = None
```

Comment:
This line looks redundant... this gets set with the next line.

Also do we want to have no figure?